ansible/README.md

```diff
@@ -14,23 +14,23 @@ can be real hardware, VMs, things in a public cloud, etc. Anything that you can
 
 ### Configure inventory
 
-Add the system information gathered above into the 'inventory' file, or create a new inventory file for the cluster.
+Add the system information gathered above into the 'inventory' file, or create a new inventory file for the cluster. Place the 'inventory' file into the `./inventory` directory.
```

Comment:
So the file is called `inventory`?

Better: ... into a file called `inventory` in the `./inventory` directory, or create a new one for the cluster.

Also use back-ticks for verbatim names.